test/unit/mdc-ripple/foundation-activation.test.js

```diff
@@ -234,6 +234,26 @@ testFoundation('adds activation classes on programmatic activation', ({foundatio
   td.verify(adapter.addClass(cssClasses.FG_ACTIVATION));
 });
 
+testFoundation('programmatic activation immediately after interaction', ({foundation, adapter, mockRaf}) => {
```

Comment:
A more descriptive name would be helpful (something like "programmatic activation that occurs immediately after interaction adds the FG activation class")

Reply:
If I make the description any longer it will overflow our maximum line length, and that's usually where I draw the line...

Reply:
Hahaha fair enough :) Approved!